### DISCUSSION TITLE
SWCOMPONENT now has no name?

### DISCUSSION THREAD

**Discussion by GregIthaca** [ge-high-assurance/RACK#317] [Q&A]
Several properties were dropped in the change from COMPONENT to SWCOMPONENT, `name` isn't included in the list of dropped attributes described in the [wiki change log](https://github.com/ge-high-assurance/RACK/wiki/RACK-v5.0-Ontology-Changelog).
If the intention is for this to start using `title` instead of `name` then perhaps this should be mentioned in the list of changed properties?

**Comment by AbhaMoitra** [comment]
In the "wiki change log" that you mention, we do have the following. Note that "title" was present from before and gets inherited by various classes. So, we listed the places where we explicitly had "name" which was dropped. Please let us know if there were other places that were affected by this change?

![image](https://user-images.githubusercontent.com/28540315/110652506-6ed73680-818a-11eb-80f9-bb6b350e0d5c.png)

**Reply by GregIthaca** [reply]
I think it would be helpful for COMPONENT to be added to this list, with a comment "replaced by SWCOMPONENT" - then it would be clear that `title` should be used for SWCOMPONENT in place of `name` in COMPONENT.
